modules/utils/include/hephaestus/utils/format/format.h

```diff
@@ -30,11 +29,10 @@ template <ArrayOrVectorType T>
 [[nodiscard]] inline auto toString(const T& vec) -> std::string {
   std::stringstream ss;
 
-  const auto indices = std::views::iota(0);
-  const auto indexed_vec = ranges::views::zip(indices, vec);
-
-  for (const auto& [index, value] : indexed_vec) {
-    const std::string str = fmt::format("  Index: {}, Value: {}\n", index, value);
+  std::size_t idx = 0;
+  for (const auto& value : vec) {
+    const std::string str = fmt::format("  Index: {}, Value: {}\n", idx, value);
+    ++idx;
```

Comment:
range/zip not supported by clang-18

Reply:
Needs checking. I think I managed to build this locally when working on the upstream PR

Reply:
pretty sure it doesn't work :)